sound/soc/sof/ipc4-priv.h

```diff
@@ -24,40 +24,55 @@ enum sof_ipc4_mtrace_type {
 	SOF_IPC4_MTRACE_INTEL_CAVS_2,
 };
 
+/**
+ * struct sof_ipc4_fw_module - IPC4 module info
+ * @sof_man4_module : Module info
+ * @m_ida: Module instance identifier
+ * @bss_size: Module object size
+ * @private: Module private data
+ */
+struct sof_ipc4_fw_module {
+	struct sof_man4_module man4_module_entry;
+	struct ida m_ida;
```

Comment:
this is very confusing, the m_ida is clearly related to a module instance but the bss_size would be common to all instances of that module, no? 

Or is the term 'module instance' an oversight, this is just the ID for a specific module that could be instantiated multiple times, e.g. on different pipelines?

Reply:
Yes, the later. One module which can be instantiated multiple times.
Note: I just moved the struct and comment w/o modification up...